nvflare/fuel/utils/pipe/cell_pipe.py

```diff
@@ -18,6 +18,7 @@
 import time
 from typing import Tuple, Union
 
+from nvflare.apis.fl_constant import SystemVarName
```

Comment:
Why is this file modified? It doesn't seem to be related to ScriptExecutor.

Reply:
this is needed for the launch_external_process option of ScriptExecutor